.github/workflows/deploy-review.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Deploy review environment"
+on:
+  pull_request:
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+    environment: 
+      name: review
+      url: https://${{ steps.slug.outputs.slug }}.technicie.nl/
+    steps:
+    - uses: actions/checkout@v2.4.0
+    - uses: cachix/install-nix-action@v16
+    - uses: cachix/cachix-action@v10
+      with:
+        name: svthalia
+        authToken: '${{ secrets.CACHIX_AUTH_TOKEN_SVTHALIA }}'
+    - name: Create slug from ref name
+      id: slug
+      run: echo "::set-output name=slug::$(echo \"${GITHUB_REF#refs/heads/}\" | iconv -t ascii//TRANSLIT | sed -E 's/[^a-zA-Z0-9]+/-/g' | sed -E 's/^-+|-+$//g' | tr A-Z a-z)"
```

Comment:
Should this not contain 'review' in one way or another? Or use a subdomain like x.review.technicie.nl so that branch names cannot override actual subdomains like blog.technicie.nl

Reply:
It's not actually the branch names, but https://refs-pull-2141-merge.technicie.nl. Good idea to add "review" as well though